src/Apps/Collect/Components/ArtworkGrid/index.tsx

```diff
@@ -2,6 +2,8 @@ import { Sans } from "@artsy/palette"
 import { ArtworkGrid_viewer } from "__generated__/ArtworkGrid_viewer.graphql"
 import { FilterState } from "Apps/Collect/FilterState"
 import { ContextConsumer } from "Artsy"
+import { Range } from "rc-slider"
+import "rc-slider/assets/index.css"
```

Comment:
Without this, the CSS isn't brought in and the slider doesn't even appear visible. With this line, it looks like something:

<img width="333" alt="screen shot 2018-09-05 at 8 28 38 pm" src="https://user-images.githubusercontent.com/1457859/45128114-4a79e980-b14a-11e8-809e-fe5fd2e9b863.png">

Reply:
we could wrap the whole stylesheet (https://github.com/react-component/slider/blob/master/assets/index.less) into a styled component and just convert the LESS in css-in-js